app/install/dcos.md

```diff
@@ -68,58 +68,237 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
     $ dcos package install cassandra
     ```
 
-    For PostgreSQL, use the `postgres.json` file from the kong-dist-dcos 
-    repo to deploy a PostgreSQL instance in the cluster:
+    For PostgreSQL, use the `postgresql` package with following option:
```

Comment:
I'd replace "with following option" with "with the following configuration" or "with the following options"